web_widget_timer/README.rst

```diff
@@ -0,0 +1,83 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
can you update according to new readme policy? https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst